### DISCUSSION TITLE
post/example

### DISCUSSION THREAD

**Discussion by giscus** [bepyan/bepyan.me.v2#2] [General]
# post/example

https://bepyan.me/post/example

**Comment by kyh196201** [comment]
평안님 글 예시랑 블로그 디자인이 너무 맘에 들어서 그러는데, 참고해서 만들어도 될까요??

**Reply by bepyan** [reply]
그럼요! 상업적인 용도가 아니라면 출처만 한번 언급해주시고 맘껏 활용하셔도 괜찮습니다!

**Reply by kyh196201** [reply]
감사합니다 ㅎㅎ !!

**Comment by LimSB-dev** [comment]
평안님 말씀처럼 글이 정말 편안하게 다왔습니다.
게시글을 읽는 내내 부담없이 정독하게 되네요.

저도 마크다운을 사용해 블로그 게시글을 작성하고 있었는데 많이 배우고 갑니다.

**Reply by bepyan** [reply]
공감이 되어 뿌듯하네요 ㅎㅎ 댓글 남겨주셔서 감사합니다!

**Comment by Voyager003** [comment]
안녕하세요 평안님. 
블로그가 너무 깔끔하고 맘에 드는데 개인 블로그로 참고해서 만들어도 될까요? (상업적 용도 X)

**Reply by bepyan** [reply]
그럼요! 소스코드도 열려 있으니 참고하셔도 좋습니다!
https://github.com/bepyan/bepyan.me.v2

**Reply by Voyager003** [reply]
감사합니다!
